src/test/java/bio/terra/service/duos/DuosClientTest.java

```diff
@@ -209,24 +220,11 @@ public void testConvertDuosExToDataRepoEx() {
   }
 
   // Utilities for examining HttpEntity parameters
```

Comment:
I broke out these utilities into their own class to reuse in the new `RawlsClientTest`.  While I was modifying `DuosClientTest` I also converted it to JUnit 5.